core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -445,6 +445,29 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
     }
   }
 
+  test("SPARK-33084: add jar when path contains comma") {
```

Comment:
For a negative test case, could you be more specific? Otherwise, this test case name can be misleading.
```scala
- test("SPARK-33084: add jar when path contains comma") {
+ test("SPARK-33084: add jar should fail when path contains comma") {
```